src/onnx/parse_qlinearadd.cpp

```diff
@@ -133,6 +134,21 @@ struct parse_qlinearadd : op_parser<parse_qlinearadd>
         const auto& in_zero_pt_b = args[5];
         auto dquant_b = bcast_qdq_instr("dequantizelinear", in_b, in_scale_b, in_zero_pt_b, info);
 
+        auto sh_a = in_a->get_shape();
+        auto sh_b = in_b->get_shape();
+
+        if(sh_a != sh_b)
+        {
+            auto common_lens = compute_broadcasted_lens(sh_a.lens(), sh_b.lens());
+            if(sh_a.lens() != common_lens)
+                dquant_a = info.add_instruction(
+                    migraphx::make_op("multibroadcast", {{"out_lens", common_lens}}), dquant_a);
+
+            if(sh_b.lens() != common_lens)
+                dquant_b = info.add_instruction(
+                    migraphx::make_op("multibroadcast", {{"out_lens", common_lens}}), dquant_b);
+        }
```

Comment:
I think here you can use `add_common_op` here

Reply:
@umangyadav see comment above

Reply:
Sure. Will do shortly. Anything else, @umangyadav & @kahmed10 ..?   Thanks.

Reply:
Done.